projects/ngx-prx-styleguide/src/lib/datepicker/datepicker.component.ts

```diff
@@ -32,6 +32,11 @@ export class DatepickerComponent implements AfterViewInit {
         const pickerDate = this.UTC ? this.pickerUTCOffset(this._date) : this._date;
         this.picker.setDate(pickerDate);
       }
+    } else if (!value) {
```

Comment:
we could initialize a date field as empty but we couldn't set it to empty with Publish's Discard button